graphics/graphite2/Portfile

```diff
@@ -1,11 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 PortGroup           muniversal 1.0
+PortGroup           github 1.0
 
+github.setup        silnrsi graphite 1.3.12
 name                graphite2
-version             1.3.9
 categories          graphics
 platforms           darwin
 maintainers         ryandesign openmaintainer
```

Comment:
This should probably be fixed to

`maintainers         {@ryandesign ryandesign} openmaintainer`